.readthedocs.yml

```
@@ -5,6 +5,7 @@ build:
   tools:
     python: "3"
   commands:
+    - export LANGUAGE=$(./convert_language_tag.py $READTHEDOCS_LANGUAGE)
```

Perhaps make the call explicit?
```suggestion
    - export LANGUAGE=$(python convert_language_tag.py $READTHEDOCS_LANGUAGE)
```